server/app/filters/RecordCookieSizeFilter.java

```diff
@@ -0,0 +1,33 @@
+package filters;
+
+import io.prometheus.client.Histogram;
+import play.mvc.EssentialAction;
+import play.mvc.EssentialFilter;
+import play.routing.HandlerDef;
+import play.routing.Router;
+
+public class RecordCookieSizeFilter extends EssentialFilter {
```

Comment:
**suggestion (blocking):** Please add javadoc and make this class final

Reply:
Done